spec/features/users/mygarden/delete_plants_spec.rb

```diff
@@ -9,30 +9,59 @@
                             name: 'gardenthattesting',
                             zip_code: '02300',
                             google_token: ENV['TEST_USER_GOOGLE_TOKEN'],
-                            google_refresh_token: ENV['TEST_USER_GOOGLE_REFRESH_TOKEN']
+                            google_refresh_token: ENV['TEST_USER_GOOGLE_REFRESH_TOKEN'],
+                            calendar_id: '5n59q0ueh4202i4mlstfbqejgc@group.calendar.google.com'
                           )
       allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user1)
 			@garden = Garden.create(name: 'Garden 1', user: @user1)
-      @plant1 = Plant.new(name: 'tomato', image: "https://www.almanac.com/sites/default/files/image_nodes/tomatoes_helios4eos_gettyimages-edit.jpeg")
-      @plant2 = Plant.new(name: 'basil', image: "https://www.almanac.com/sites/default/files/image_nodes/tomatoes_helios4eos_gettyimages-edit.jpeg")
-      @garden.plants << @plant1
-      @garden.plants << @plant2
     end
 
-    it "I can delete the plants in my garden" do 
+    it "I can delete the plants in my garden", :vcr do 
 
+      visit '/'
+			fill_in "search", with: "tomato"
+			click_on "Search"
+			click_on "Tomato"
+      click_on "Add to MyGarden"
+      click_link 'Garden 1'
 
-      expect(GardenPlant.where("garden_id= #{@garden.id} AND plant_id= #{@plant1.id}")).to_not be_empty
-      expect(GardenPlant.where("garden_id= #{@garden.id} AND plant_id= #{@plant2.id}")).to_not be_empty
+      visit '/'
+			fill_in "search", with: "carrot"
+			click_on "Search"
+			click_on "Carrot"
+			click_on "Add to MyGarden"
+      click_link 'Garden 1'
       
+      events1 = CalendarService.list_events(@user1)
+      list_events1 = events1.map {|event| event[:name]}
+
+      event_tomato1 = list_events1.find_all {|event| event.include?('Tomato')} 
+      event_carrot1 = list_events1.find_all {|event| event.include?('Carrot')} 
```

Comment:
Add the test that `events` are created and deleted as they should be when adding/ removing `plants` to/from a `garden`